core/runner/pom.xml

```diff
@@ -141,6 +129,17 @@
           </execution>
         </executions>
       </plugin>
+      <!-- overwrites configuration from parent pom to disable SCR plugin -->
```

Comment:
We probably should add `<!-- ToDo: remove me when all felix dependencies are removed -->` here.